src/test/java/us/kbase/test/auth2/lib/identity/IdentityProviderConfigTest.java

```diff
@@ -61,7 +61,7 @@ public void goodInput() throws Exception {
 		assertThat("incorrect link redirect URL", c.getLinkRedirectURL("env2"),
 				is(new URL("https://linkredirect2.com")));
 		assertThat("incorrect login redirect URL", c.getLoginRedirectURL(),
-				is(new URL("https://loginredirect.com")));
+				is(new URL("https://fakeloginredirect.com")));
```

Comment:
Perhaps we should use a fake KBase subdomain for stable not-hanging not-existingness?